ui/desktop/src/utils/settings.ts

```diff
@@ -9,13 +9,18 @@ export interface EnvToggles {
 }
 
 export type SchedulingEngine = 'builtin-cron' | 'temporal';
+export type TunnelMode = 'lapstone' | 'tailscale';
 
 export interface Settings {
   envToggles: EnvToggles;
   showMenuBarIcon: boolean;
   showDockIcon: boolean;
   schedulingEngine: SchedulingEngine;
   enableWakelock: boolean;
+  tunnelSecret?: string;
+  tunnelAgentId?: string;
+  tunnelMode?: TunnelMode;
+  tunnelAutoStart?: boolean;
```

Comment:
we should not be using this - settings should go through the server, saving the settings here will potentially miss changes we made since.

Reply:
yes, that makes sense. I was thinking goose-server didn't need to care about this, but could shift logic there.